### ISSUE TITLE
Make:factory is missing

### ISSUE THREAD

**Issue by segunmicheal27** [flipboxstudio/lumen-generator#62]
We will be so glad if Make:factory can be added

**Comment by zroob** [comment]
Your package is conflicting because of the nikic/php-parser
```
  Problem 1
    - Conclusion: don't install flipbox/lumen-generator 6.0.1
    - Conclusion: remove nikic/php-parser v4.2.5
    - Installation request for flipbox/lumen-generator ^6.0 -> satisfiable by flipbox/lumen-generator[6.0, 6.0.1].
    - Conclusion: don't install nikic/php-parser v4.2.5
    - flipbox/lumen-generator 6.0 requires classpreloader/classpreloader ^3.0 -> satisfiable by classpreloader/classpreloader[3.0.0, 3.1.0, 3.2.0, 3.2.x-dev].
    - classpreloader/classpreloader 3.0.0 requires nikic/php-parser ^1.0|^2.0 -> satisfiable by nikic/php-parser[1.x-dev, 2.x-dev, v1.0.0, v1.0.0beta1, v1.0.0beta2, v1.0.1, v1.0.2, v1.1.0, v1.2.0, v1.2.1, v1.2.2, v1.3.0, v1.4.0, v1.4.1, v2.0.0, v2.0.0alpha1, v2.0.0beta1, v2.0.1, v2.1.0, v2.1.1].
    - classpreloader/classpreloader 3.1.0 requires nikic/php-parser ^1.0|^2.0|^3.0 -> satisfiable by nikic/php-parser[1.x-dev, 2.x-dev, 3.x-dev, v1.0.0, v1.0.0beta1, v1.0.0beta2, v1.0.1, v1.0.2, v1.1.0, v1.2.0, v1.2.1, v1.2.2, v1.3.0, v1.4.0, v1.4.1, v2.0.0, v2.0.0alpha1, v2.0.0beta1, v2.0.1, v2.1.0, v2.1.1, v3.0.0, v3.0.0alpha1, v3.0.0beta1, v3.0.0beta2, v3.0.1, v3.0.2, v3.0.3, v3.0.4, v3.0.5, v3.0.6, v3.1.0, v3.1.1, v3.1.2, v3.1.3, v3.1.4, v3.1.5].
    - classpreloader/classpreloader 3.2.0 requires nikic/php-parser ^1.0|^2.0|^3.0 -> satisfiable by nikic/php-parser[1.x-dev, 2.x-dev, 3.x-dev, v1.0.0, v1.0.0beta1, v1.0.0beta2, v1.0.1, v1.0.2, v1.1.0, v1.2.0, v1.2.1, v1.2.2, v1.3.0, v1.4.0, v1.4.1, v2.0.0, v2.0.0alpha1, v2.0.0beta1, v2.0.1, v2.1.0, v2.1.1, v3.0.0, v3.0.0alpha1, v3.0.0beta1, v3.0.0beta2, v3.0.1, v3.0.2, v3.0.3, v3.0.4, v3.0.5, v3.0.6, v3.1.0, v3.1.1, v3.1.2, v3.1.3, v3.1.4, v3.1.5].
    - classpreloader/classpreloader 3.2.x-dev requires nikic/php-parser ^1.0|^2.0|^3.0 -> satisfiable by nikic/php-parser[1.x-dev, 2.x-dev, 3.x-dev, v1.0.0, v1.0.0beta1, v1.0.0beta2, v1.0.1, v1.0.2, v1.1.0, v1.2.0, v1.2.1, v1.2.2, v1.3.0, v1.4.0, v1.4.1, v2.0.0, v2.0.0alpha1, v2.0.0beta1, v2.0.1, v2.1.0, v2.1.1, v3.0.0, v3.0.0alpha1, v3.0.0beta1, v3.0.0beta2, v3.0.1, v3.0.2, v3.0.3, v3.0.4, v3.0.5, v3.0.6, v3.1.0, v3.1.1, v3.1.2, v3.1.3, v3.1.4, v3.1.5].
    - Can only install one of: nikic/php-parser[1.x-dev, v4.2.5].
    - Can only install one of: nikic/php-parser[2.x-dev, v4.2.5].
    - Can only install one of: nikic/php-parser[3.x-dev, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.2.0, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.2.1, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.2.2, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.3.0, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.4.0, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.4.1, v4.2.5].
    - Can only install one of: nikic/php-parser[v2.0.0, v4.2.5].
    - Can only install one of: nikic/php-parser[v2.0.0alpha1, v4.2.5].
    - Can only install one of: nikic/php-parser[v2.0.0beta1, v4.2.5].
    - Can only install one of: nikic/php-parser[v2.0.1, v4.2.5].
    - Can only install one of: nikic/php-parser[v2.1.0, v4.2.5].
    - Can only install one of: nikic/php-parser[v2.1.1, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.0.0, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.0.0alpha1, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.0.0beta1, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.0.0beta2, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.0.1, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.0.2, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.0.3, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.0.4, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.0.5, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.0.6, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.1.0, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.1.1, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.1.2, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.1.3, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.1.4, v4.2.5].
    - Can only install one of: nikic/php-parser[v3.1.5, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.0.0, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.0.0beta1, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.0.0beta2, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.0.1, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.0.2, v4.2.5].
    - Can only install one of: nikic/php-parser[v1.1.0, v4.2.5].
    - Installation request for nikic/php-parser (locked at v4.2.5) -> satisfiable by nikic/php-parser[v4.2.5].
```
My composerjson:
```
"require": {
        "php": ">=7.1.3",
        "aunbox/gsx-grant": "^1.0.0",
        "dingo/api": "^2.0",
        "guzzlehttp/guzzle": "^6.4",
        "illuminate/mail": "5.8.*",
        "illuminate/redis": "^5.4",
        "jeremeamia/superclosure": "^2.4",
        "laravel/lumen-framework": "5.8.*",
        "yiisoft/cache": "^3.0@dev"
    },
    "require-dev": {
        "fzaninotto/faker": "^1.4",
        "phpunit/phpunit": "^7.0",
        "mockery/mockery": "^1.0"
    },
```

**Comment by joelhy** [comment]
> Your package is conflicting because of the nikic/php-parser
> 
> ```
>   Problem 1
>     - Conclusion: don't install flipbox/lumen-generator 6.0.1
>     - Conclusion: remove nikic/php-parser v4.2.5
>     - Installation request for flipbox/lumen-generator ^6.0 -> satisfiable by flipbox/lumen-generator[6.0, 6.0.1].
>     - Conclusion: don't install nikic/php-parser v4.2.5
>     - flipbox/lumen-generator 6.0 requires classpreloader/classpreloader ^3.0 -> satisfiable by classpreloader/classpreloader[3.0.0, 3.1.0, 3.2.0, 3.2.x-dev].
>     - classpreloader/classpreloader 3.0.0 requires nikic/php-parser ^1.0|^2.0 -> satisfiable by nikic/php-parser[1.x-dev, 2.x-dev, v1.0.0, v1.0.0beta1, v1.0.0beta2, v1.0.1, v1.0.2, v1.1.0, v1.2.0, v1.2.1, v1.2.2, v1.3.0, v1.4.0, v1.4.1, v2.0.0, v2.0.0alpha1, v2.0.0beta1, v2.0.1, v2.1.0, v2.1.1].
>     - classpreloader/classpreloader 3.1.0 requires nikic/php-parser ^1.0|^2.0|^3.0 -> satisfiable by nikic/php-parser[1.x-dev, 2.x-dev, 3.x-dev, v1.0.0, v1.0.0beta1, v1.0.0beta2, v1.0.1, v1.0.2, v1.1.0, v1.2.0, v1.2.1, v1.2.2, v1.3.0, v1.4.0, v1.4.1, v2.0.0, v2.0.0alpha1, v2.0.0beta1, v2.0.1, v2.1.0, v2.1.1, v3.0.0, v3.0.0alpha1, v3.0.0beta1, v3.0.0beta2, v3.0.1, v3.0.2, v3.0.3, v3.0.4, v3.0.5, v3.0.6, v3.1.0, v3.1.1, v3.1.2, v3.1.3, v3.1.4, v3.1.5].
>     - classpreloader/classpreloader 3.2.0 requires nikic/php-parser ^1.0|^2.0|^3.0 -> satisfiable by nikic/php-parser[1.x-dev, 2.x-dev, 3.x-dev, v1.0.0, v1.0.0beta1, v1.0.0beta2, v1.0.1, v1.0.2, v1.1.0, v1.2.0, v1.2.1, v1.2.2, v1.3.0, v1.4.0, v1.4.1, v2.0.0, v2.0.0alpha1, v2.0.0beta1, v2.0.1, v2.1.0, v2.1.1, v3.0.0, v3.0.0alpha1, v3.0.0beta1, v3.0.0beta2, v3.0.1, v3.0.2, v3.0.3, v3.0.4, v3.0.5, v3.0.6, v3.1.0, v3.1.1, v3.1.2, v3.1.3, v3.1.4, v3.1.5].
>     - classpreloader/classpreloader 3.2.x-dev requires nikic/php-parser ^1.0|^2.0|^3.0 -> satisfiable by nikic/php-parser[1.x-dev, 2.x-dev, 3.x-dev, v1.0.0, v1.0.0beta1, v1.0.0beta2, v1.0.1, v1.0.2, v1.1.0, v1.2.0, v1.2.1, v1.2.2, v1.3.0, v1.4.0, v1.4.1, v2.0.0, v2.0.0alpha1, v2.0.0beta1, v2.0.1, v2.1.0, v2.1.1, v3.0.0, v3.0.0alpha1, v3.0.0beta1, v3.0.0beta2, v3.0.1, v3.0.2, v3.0.3, v3.0.4, v3.0.5, v3.0.6, v3.1.0, v3.1.1, v3.1.2, v3.1.3, v3.1.4, v3.1.5].
>     - Can only install one of: nikic/php-parser[1.x-dev, v4.2.5].
>     - Can only install one of: nikic/php-parser[2.x-dev, v4.2.5].
>     - Can only install one of: nikic/php-parser[3.x-dev, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.2.0, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.2.1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.2.2, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.3.0, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.4.0, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.4.1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v2.0.0, v4.2.5].
>     - Can only install one of: nikic/php-parser[v2.0.0alpha1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v2.0.0beta1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v2.0.1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v2.1.0, v4.2.5].
>     - Can only install one of: nikic/php-parser[v2.1.1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.0.0, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.0.0alpha1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.0.0beta1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.0.0beta2, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.0.1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.0.2, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.0.3, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.0.4, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.0.5, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.0.6, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.1.0, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.1.1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.1.2, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.1.3, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.1.4, v4.2.5].
>     - Can only install one of: nikic/php-parser[v3.1.5, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.0.0, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.0.0beta1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.0.0beta2, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.0.1, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.0.2, v4.2.5].
>     - Can only install one of: nikic/php-parser[v1.1.0, v4.2.5].
>     - Installation request for nikic/php-parser (locked at v4.2.5) -> satisfiable by nikic/php-parser[v4.2.5].
> ```
> 
> My composerjson:
> 
> ```
> "require": {
>         "php": ">=7.1.3",
>         "aunbox/gsx-grant": "^1.0.0",
>         "dingo/api": "^2.0",
>         "guzzlehttp/guzzle": "^6.4",
>         "illuminate/mail": "5.8.*",
>         "illuminate/redis": "^5.4",
>         "jeremeamia/superclosure": "^2.4",
>         "laravel/lumen-framework": "5.8.*",
>         "yiisoft/cache": "^3.0@dev"
>     },
>     "require-dev": {
>         "fzaninotto/faker": "^1.4",
>         "phpunit/phpunit": "^7.0",
>         "mockery/mockery": "^1.0"
>     },
> ```

@ztpan

Cannot reproduce this problem.
Please do not mess with other's issue. 
You can create another new issue.

**Comment by joelhy** [comment]
Fixed in v6.0.2
